deploy/examples/create-external-cluster-resources.py

```diff
@@ -617,10 +628,59 @@ def endpoint_dial(self, endpoint_str, ip_type, timeout=3, cert=None):
             ("-1"),
         )
 
+    def parse_config(self, config_file):
+        config = configparser.ConfigParser()
+        config.read(config_file)
+
+        configConfigurations = [
+            "verbose",
+            "ceph-conf",
+            "keyring",
+            "run-as-user",
+            "cluster-name",
+            "k8s-cluster-name",
+            "namespace",
+            "rgw-pool-prefix",
+            "restricted-auth-permission",
+            "v2-port-enable",
+            "format",
+            "output",
+            "cephfs-filesystem-name",
+            "cephfs-metadata-pool-name",
+            "cephfs-data-pool-name",
+            "rbd-data-pool-name",
+            "alias-rbd-data-pool-name",
+            "rgw-endpoint",
+            "rgw-tls-cert-path",
+            "rgw-skip-tls",
+            "monitoring-endpoint",
+            "monitoring-endpoint-port",
+            "skip-monitoring-endpoint",
+            "rbd-metadata-ec-pool-name",
+            "dry-run",
+            "rados-namespace",
+            "subvolume-group",
+            "rgw-realm-name",
+            "rgw-zone-name",
+            "rgw-zonegroup-name",
+            "topology-pool",
+            "topology-failure-domain-label",
+            "topology-failure-domain-values",
+            "upgrade",
+        ]
```

Comment:
Can these be defined from the existing CLI arguments? 

If we have to copy CLI flag names to this section, it will make future maintenance harder and more likely to have bugs. For example, it would be very easy for an upstream user to add a contribution to add a CLI flag and forget the associated config, or vice versa.

Could we instead process the `config_group` args to generate this list in code?